packages/@uppy/companion/Dockerfile

```
@@ -9,13 +9,13 @@ WORKDIR /app
 ADD package.json package-*.json yarn.* /tmp/
 RUN cd /tmp && apk --update add  --virtual native-dep \
   make gcc g++ python3 libgcc libstdc++ git && \
-  corepack yarn install && \
+  npm install && \
```

Moved to https://github.com/transloadit/uppy/pull/3355.
```suggestion
  corepack yarn install && \
```